csharp/src/Ice/README.md

```diff
@@ -29,55 +30,50 @@ module VisitorCenter
 ```csharp
 // Client application
 
-// Slice module VisitorCenter in Greeter.ice maps to C# namespace VisitorCenter.
 using VisitorCenter;
 
-// Create an Ice communicator.
 await using var communicator = new Ice.Communicator(ref args);
 
-// GreeterPrx is a class generated by the Slice compiler. We create a proxy from a communicator and a "stringified
```

Comment:
We want the sample code to be limited to about 80 columns to avoid horizontal scroll bars in the NuGet rendering.